test/__snapshots__/to-contain.spec.js.snap

```diff
@@ -1,14 +1,63 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`to-contain <ReactElement> fails when the actual does not match the expected 1`] = `
-"expected <Fixture /> to contain with all wrappers <User id={4} />
+"expected
+<Fixture>
+  <div>
+    <ul>
+      <li>
+        <User id={1} name=\\"John\\">
+          <span className=\\"user\\" id=\\"user-1\\">User 1</span>
+        </User>
+      </li>
+      <li>
+        <User id={2} name=\\"Doe\\">
+          <span className=\\"user\\" id=\\"user-2\\">User 2</span>
+        </User>
+        <User id={3} name=\\"Mark\\">
+          <span className=\\"user\\" id=\\"user-3\\">User 3</span>
+        </User>
+      </li>
+    </ul>
+  </div>
+</Fixture>
+to contain <User id={4} />
 
 the best match was
 <Fixture // should be <User
    // missing id={4}
 />"
```

Comment:
This seems to be a bug in htmllike, if I call children on the wrapper then a get a really nice diff.